### ISSUE TITLE
Prevent concurrent conflicting transactions by incrementing a local revision number

### ISSUE THREAD

**Issue by n1k0** [mozilla-services/cliquetis#34]

**Comment by michielbdejong** [comment]
Makes sense! Already needed for when a user has the same app open in multiple browser tabs, but becomes even more important when using workers. It's a pity that IndexedDB (unlike localStorage) doesn't implement something like StorageEvent, which would have been ideal for this, since it also gives a way to update the UI as soon as a worker (or other browser tab) changes something. In the absence of that, local revision numbers seem like a good second option.
